python/cudf/cudf/core/column/struct.py

```diff
@@ -153,6 +156,36 @@ def _with_type_metadata(self: StructColumn, dtype: Dtype) -> StructColumn:
 
         return self
 
+    def serialize(self) -> Tuple[dict, list]:
+        header, frames = super().serialize()
+        header["dtype"] = self.dtype.serialize()
+        header["size"] = self.size
+
+        header["sub-frame-offset"] = len(frames)
+        sub_headers = []
+        for item in self.children:
+            sheader, sframes = item.serialize()
+            sub_headers.append(sheader)
+            frames.extend(sframes)
+
+        header["sub-headers"] = sub_headers
+        header["frame_count"] = len(frames)
+        return header, frames
+
+    @classmethod
+    def deserialize(cls, header: dict, frames: list) -> StructColumn:
+        header["dtype"] = cudf.StructDtype.deserialize(*header["dtype"])
```

Comment:
```suggestion
        header["dtype"] = self.dtype.__class__.deserialize(*header["dtype"])
```

Reply:
This doesn't work, `cls.dtype` is a `@property` in `cdef class Column` not a static variable :/
See: https://gpuci.gpuopenanalytics.com/job/rapidsai/job/gpuci/job/cudf/job/prb/job/cudf-gpu-test/CUDA=11.5,GPU_LABEL=driver-495,LINUX_VER=ubuntu20.04,PYTHON=3.9/7833/console